### PR TITLE
Create GA announcement, delete beta banner (don't merge until GA)!

### PR DESCRIPTION
**Pull request by austin-schaefer**
### Tell us why

The site is GA now!

### Anything else you'd like to share?

Can we set this PR to merge just before the new site goes live?

### CODE REVIEW

**Review by jerelmiller — APPROVED** [review]

**Comment by zstix** [comment]
@austin-schaefer announcement banners only display when the current date is within the supplied date range. We should be alright to merge this whenever, but I'm happy to hold off if you would like.

File: src/announcements/docs-ga-banner.mdx
Comment (jerelmiller) [inline src/announcements/docs-ga-banner.mdx:2]:
```suggestion
startDate: 2021-03-01
```

Just realized, this has the wrong year!

**Review by jerelmiller — CHANGES_REQUESTED** [review]

**Review by jerelmiller — APPROVED** [review]
As @zstix said, we can merge this whenever. It won't go live until the `startDate`, so we are safe to merge it now.

**Comment by austin-schaefer** [comment]
@jerelmiller @zstix : Well won't it go live at 12:01 am on March 1, hours ahead of scheduled GA?

**Comment by jerelmiller** [comment]
@austin-schaefer You're right, that is true. This does use an ISO 8601 date format, so we _could_ in theory use a time as well here. Meh, let's just wait until Monday.

**Pull request merged**
